db/seeds.rb

```diff
@@ -5,3 +5,36 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+
+
+  category1 = Category.create(title: "Comedies")
+  category2 = Category.create(title: "Dramas")
+  category3 = Category.create(title: "Reality")
+
+  4.times do 
+    Video.create(title: 'Family Guy', description: 'A show', 
+                  small_cover_url: '/tmp/family_guy.jpg', 
+                  large_cover_url: '/tmp/monk_large.jpg',
+                  category: category1)
+  end
+  
+  3.times do 
+    Video.create(title: 'Futurama', description: 'A show', 
+                  small_cover_url: '/tmp/futurama.jpg', 
+                  large_cover_url: '/tmp/monk_large.jpg',
+                  category: category1)
+  end
+
+  2.times do 
+    Video.create(title: 'Family Guy', description: 'A show', 
+                  small_cover_url: '/tmp/family_guy.jpg', 
+                  large_cover_url: '/tmp/monk_large.jpg',
+                  category: category2)
+  end
+  
+  3.times do 
+    Video.create(title: 'Futurama', description: 'A show', 
```

Comment:
You should indent hash keys at the same level and aligned. Also `create!` should be used as it will notify you when something is not valid, while normal `create` won't.